JavaScript/JavaScriptSDK/AppInsights.ts

```diff
@@ -178,8 +205,9 @@ module Microsoft.ApplicationInsights {
          * @param   url   String - a relative or absolute URL that identifies the page or other item. Defaults to the window location.
          * @param   properties  map[string, string] - additional data used to filter pages and metrics in the portal. Defaults to empty.
          * @param   measurements    map[string, number] - metrics associated with this page, displayed in Metrics Explorer on the portal. Defaults to empty.
+         * @param   EndDate A date that identifies the original end date that will override the current date. Defaults to the current date.
          */
-        public stopTrackPage(name?: string, url?: string, properties?: Object, measurements?: Object) {
+        public stopTrackPage(name?: string, url?: string, properties?: Object, measurements?: Object, endTime?: number) {
             try {
                 if (typeof name !== "string") {
```

Comment:
The same here, both variables will be set in `stopTrackPageInternal`. No need to duplicate the code.